### ISSUE TITLE
Removes ctypes install dependencies from setup.py

### ISSUE THREAD

**Issue by jacksilv** [dagwieers/vmguestlib#6]
Hello,
I was testing your tool, but I encountered an issue.
As the newest versions of python (> 2.5) includes the ctypes package, the installation of this library fails when you use pip.

`No local packages or working download links found for ctypes
error: Could not find suitable distribution for Requirement.parse('ctypes')`

The solution would be simply to remove this dependency. I  tested it and it works for me. I can provide a PR if you want.

**Comment by alex-roman** [comment]
@jacksilv can you share your solution please. I have ctypes removed from setup and got my vmguestlib broken. Thanks.

**Comment by jacksilv** [comment]
@alex-roman 
I am using Python 2.7.10
On a clean virtualenv I simply applied this diff:

`diff --git a/setup.py b/setup.py
index 6ee9ea4..ab6e73c 100755
--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,6 @@ if __name__ == '__main__':
         download_url='http://github.com/dagwieers/vmguestlib/archive/0.1.2.tar.gz',
         description='Python API for interacting with VMware\'s VMGuestLib SDK',
         license = 'GPLv2',
-        install_requires=['ctypes', ],
         py_modules = ['vmguestlib', ],
         scripts=[ 'vmguest-stats', ],
         keywords = ['Virtual', 'vmware', 'ESX', 'ESXi', 'VMGuestLib', 'SDK', 'API'],`

and I run

> python setup.py install

What kind of error did you get?

**Comment by alex-roman** [comment]
@jacksilv thanks bud! All good now.

**Comment by alex-roman** [comment]
```
pip install vmguestlib
Downloading/unpacking vmguestlib
  Downloading vmguestlib-0.1.2.tar.gz
  Running setup.py (path:/tmp/pip_build_root/vmguestlib/setup.py) egg_info for package vmguestlib

Downloading/unpacking ctypes (from vmguestlib)
  Could not find any downloads that satisfy the requirement ctypes (from vmguestlib)
Cleaning up...
No distributions at all found for ctypes (from vmguestlib)
Storing debug log for failure in /home/armn/.pip/pip.log
```
This also affects pip. 
Ubuntu14

**Comment by manvigill** [comment]
I am seeing the same issue on Ubuntu 16.04

**Comment by ev4ern** [comment]
+

**Comment by dagwieers** [comment]
Fixed by #8
